internal/backend/backend_retry.go

```diff
@@ -33,7 +34,16 @@ func NewRetryBackend(be restic.Backend, maxTries int, report func(string, error,
 }
 
 func (be *RetryBackend) retry(ctx context.Context, msg string, f func() error) error {
-	err := backoff.RetryNotify(f,
+	nf := func() error {
+		err := f()
+		if err != nil && errors.IsFatal(errors.Cause(err)) {
+			be.Report(msg, err, 0)
+			return backoff.Permanent(err)
```

Comment:
Why not simply use `backoff.Permanent(err)` when initially returning an error, instead of hot-patching it here?

Reply:
Didn't want to use package specific error throughout project, when there already is project specific error Fatal.